<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Kontakt</class>
 <widget class="QMainWindow" name="Kontakt">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>648</width>
    <height>633</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="12" column="1" colspan="3">
       <widget class="QLineEdit" name="uiMail2Ledit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="7" column="1" colspan="2">
       <widget class="QLineEdit" name="uiTelefon1Ledit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="16" column="1" colspan="3">
       <widget class="QLabel" name="uiVertreterAdresse1Lbl">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="21" column="1" colspan="3">
       <widget class="QPlainTextEdit" name="uiAnmPedit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="tabChangesFocus">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="9" column="1" colspan="2">
       <widget class="QLineEdit" name="uiTelefon3Ledit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLineEdit" name="uiStrasseLedit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_10">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>e-Mail:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_3">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Telefon:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="Line" name="line_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="3" column="0" colspan="4">
       <widget class="Line" name="line">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="uiTypLbl">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Typ:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="0" colspan="4">
       <widget class="Line" name="line_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="uiNachnameLbl">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Gemeinschaft/Organisation:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="14" column="0" colspan="4">
       <widget class="Line" name="line_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_5">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Telefon:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="uiVertreterLbl">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vertreter/Obmann:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="13" column="1" colspan="3">
       <widget class="QLineEdit" name="uiMail3Ledit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="19" column="1" colspan="3">
       <widget class="QLabel" name="uiVertreterMailLbl">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="3">
       <widget class="QLineEdit" name="uiNachnameLedit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_6">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Stra√üe:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_12">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>e-Mail:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>e-Mail:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="uiVornameLedit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="8" column="1" colspan="2">
       <widget class="QLineEdit" name="uiTelefon2Ledit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="21" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="topMargin">
         <number>5</number>
        </property>
        <item>
         <widget class="QLabel" name="label_11">
          <property name="font">
           <font>
            <family>Calibri</family>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Anmerkung:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item row="17" column="1" colspan="3">
       <widget class="QLabel" name="uiVertreterAdresse2Lbl">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="18" column="1" colspan="3">
       <widget class="QLabel" name="uiVertreterTelefonLbl">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
      <item row="5" column="2" colspan="2">
       <widget class="QLineEdit" name="uiOrtLedit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="11" column="1" colspan="3">
       <widget class="QLineEdit" name="uiMail1Ledit">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="15" column="1" colspan="3">
       <widget class="ContactCombo" name="uiVertreterCombo">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_8">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Telefon:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QLineEdit" name="uiPlzLedit">
        <property name="maximumSize">
         <size>
          <width>80</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="clearButtonEnabled">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_7">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Post</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="uiVornameLbl">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Vorname:</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="20" column="0" colspan="4">
       <widget class="Line" name="line_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QComboBox" name="uiTypCombo">
        <property name="font">
         <font>
          <family>Calibri</family>
          <pointsize>11</pointsize>
         </font>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>648</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ContactCombo</class>
   <extends>QComboBox</extends>
   <header>app_core.scopes.kontakt.contact_combo</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>uiTypCombo</tabstop>
  <tabstop>uiNachnameLedit</tabstop>
  <tabstop>uiVornameLedit</tabstop>
  <tabstop>uiStrasseLedit</tabstop>
  <tabstop>uiPlzLedit</tabstop>
  <tabstop>uiOrtLedit</tabstop>
  <tabstop>uiTelefon1Ledit</tabstop>
  <tabstop>uiTelefon2Ledit</tabstop>
  <tabstop>uiTelefon3Ledit</tabstop>
  <tabstop>uiMail1Ledit</tabstop>
  <tabstop>uiMail2Ledit</tabstop>
  <tabstop>uiMail3Ledit</tabstop>
  <tabstop>uiVertreterCombo</tabstop>
  <tabstop>uiAnmPedit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
